src/odemis/acq/align/spot.py

```diff
@@ -354,6 +354,11 @@ def FindGridSpots(image, repetition):
         Data array containing the greyscale image.
     repetition : tuple of ints
         Number of expected spots in (X, Y).
+    method : str, "affine" or "similarity"
```

Comment:
Instead of a "str", please create two constants (like `GRID_AFFINE`, `GRID_SIMILARITY`) and use them here.
This way, it's clearer that a it's not *any* string, and also you'll reduce chances of typos in the name.